Makefile

```diff
@@ -116,6 +116,12 @@ run: build ## Run
 build-image:
 	DOCKER_BUILDKIT=1 $(OCI_RUNTIME) build -t $(DOCKER_IMG):$(DOCKER_TAG) -f contrib/docker/Dockerfile .
 
+.PHONY: build-main-and-sha
+build-main-and-sha: build-image
```

Comment:
Should this be really `build-main-and-sha` or we can rename it to `build-image-with-commit-sha`

BTW: in general I do not like the complexity so much ... from my POV we need two images (1) a `latest` tag and (2) an image with a `version` tag . This is what I am used to from other projects ... I agree and it make sense to me that those  version tags can be temporal unless they are releases ... but the code here is a bit too complicated for me.

Reply:
This has proven helpful sometimes to the QE to be able to fallback on a recent image when the main/latest was broken

Reply:
the idea is really not complicated, it's only to rebuild the same image with a label `--label quay.expires-after=2w` and a different tag, nothing more. I wish there is a simpler way to implement it, but the simplest I found was to have this temporary dockerfile (if you can think about a simpler way, I'm all ears :) )

Reply:
Got it + now when you added the `[shortlived.Dockerfile](https://github.com/netobserv/flowlogs-pipeline/pull/129/files#diff-2ed92d81a3aa4dd87afe89557f95ec38735e3c175dcad5aabbe79595bbf9ff01)` thing it is much more understood (at least for me) ... so I am good with that